src/index.html

```diff
@@ -2,16 +2,203 @@
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML Form Task</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldSet">
+        <legend>Personal information</legend>
+        <div class="formField">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            required
+            id="surname"
+            autocomplete="off"
+              >
+        </div>
+
+        <div class="formField">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="age">How old are You? </label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            placeholder="12"
+            min="1"
+            max="100"
+            minlength="1"
+            maxlength="3"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            name="birthDate"
+            id="date"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="termsOfAgreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="termsOfAgreement"
+            id="termsOfAgreement"
+            required
+          >
+            </label>
+        </div>
+      </fieldset >
+
+      <fieldset class="fieldSet">
+        <legend>Registration</legend>
+        <div class="formField">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+              Do you love cats?
```

Comment:
2nd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
            Do you love cats?
```